stats/stats.go

```diff
@@ -78,6 +78,13 @@ func StartStatsServerWithPort(startupOpts StartupOptions, addhandlers ...func(mu
 			setLevel = zapcore.InfoLevel
 		}
 		logLevel = zap.NewAtomicLevelAt(setLevel)
+
+		logConfig := zap.NewProductionConfig()
```

Comment:
Has this issue always existed? I thought the ticket mentioned this was just for 1.7.x?

Reply:
I added the setting of log level via env variable in 1.7 but think I just missed the step of building that when it was initially added. That's likely why it's only seen for 1.7 (when users are trying to set the level via env var)